cmd/manager/main.go

```diff
@@ -228,6 +241,8 @@ func main() {
 			KlusterletWorkLister:     listersworkv1.NewManifestWorkLister(klusterletWorksInformer.GetIndexer()),
 			HostedWorkInformer:       hostedWorksInformer,
 			HostedWorkLister:         listersworkv1.NewManifestWorkLister(hostedWorksInformer.GetIndexer()),
+			KlusterletConfigInformer: klusterletconfigInformer,
```

Comment:
I did not see these fields used by any controllers or pkg in this PR.

Reply:
Will there be another PR to modify the controllers or pkg to use these?

Reply:
Yes, exactly, I try to split these into smaller PRs for better review.

Reply:
After a thought, it's better to make it a complete submit. Change status to WIP.